kani-compiler/src/codegen_cprover_gotoc/compiler_interface.rs

```diff
@@ -136,15 +141,13 @@ impl CodegenBackend for GotocCodegenBackend {
         // Print compilation report.
         print_report(&gcx, tcx);
 
-        let unsupported_features = gcx.unsupported_metadata();
-
         // perform post-processing symbol table passes
-        let passes = self.queries.get_symbol_table_passes();
-        let symtab = symtab_transformer::do_passes(gcx.symbol_table, &passes);
+        //let passes = self.queries.get_symbol_table_passes();
+        //let symtab = symtab_transformer::do_passes(gcx.symbol_table, &passes);
```

Comment:
Soo... Is there an ordering these PRs should be merged?

Reply:
Yes. I was hoping to merge the deletion of this code first so I don't need to bother about changing it since gcx is being partially moved.